app/mailers/user_mailer.rb

```diff
@@ -105,7 +105,7 @@ def claim_notification(creator_id, claimed_work_ids, is_user=false)
     I18n.with_locale(locale) do
       mail(
         to: creator.email,
-        subject: "[#{ArchiveConfig.APP_SHORT_NAME}] Works uploaded"
```

Comment:
Might be out of this PR scope. But, for my own curiosity: Is this code used in a non-AO3 context, thus the need to pass `ArchiveConfig.APP_SHORT_NAME` as a parameter each time instead of simply having AO3 "hard written" in the translation?

Reply:
I don't know about this particular email, but in general, yes. In fact, one of the people who run the archive code in a non-AO3 context notes the hardcoded AO3 text in mailers: https://squidgeworld.org/works/34491/chapters/77523